src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -1,12 +1,13 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-import { WorkAllocationTaskService } from 'src/work-allocation/services/work-allocation-task.service';
 
-import { TaskService, TaskSort } from '../../enums';
+import { TaskService, TaskSort, TaskActionIds } from '../../enums';
 import { SearchTaskRequest } from '../../models/dtos/search-task-request';
 import { Task, TaskFieldConfig, TaskSortField } from '../../models/tasks';
 import InvokedTaskAction from '../../models/tasks/invoked-task-action.model';
 import TaskServiceConfig from '../../models/tasks/task-service-config.model';
+import { WorkAllocationTaskService } from '../../services/work-allocation-task.service';
+import {InfoMessageCommService} from '../../services/info-message-comms.service';
```

Comment:
And I think this will be another lint warning as the imports also need to be alphabetical.

Reply:
Done.